docs/PacketFence_Developers_Guide.asciidoc

```diff
@@ -118,12 +118,44 @@ HTTP JSON API
 
 PacketFence provides an HTTP JSON API which exposes most of its functionality. 
 
-The full API spec documentation for the latest stable release is available from: http://petstore.swagger.io/?url=https://raw.githubusercontent.com/inverse-inc/packetfence/stable/docs/api/spec/openapi.yaml 
+The full API spec documentation for :
+
+* the latest stable release is available from: https://packetfence.org/doc/api/
+* the latest devel release is available from: https://packetfence.org/doc/api-devel/
 
 The API is exposed on 127.0.0.1 (localhost) as well as the management interface of the server for remote calls.
+You should use 9999 TCP port and a secure connection (HTTPS) to ask it.
 
 Authentication can be done using either the webservices credentials or any credentials that are valid on the web admin interface.
 
+How to use API
+~~~~~~~~~~~~~~
+
+NOTE: You can use API spec documentation mentionned above to have ready to use curl commands with "Try it out" button.
+
+First, get an authentication token with webservices credentials or an admin account:
+
+ curl -X POST "https://PF_MANAGEMENT_IP:9999/api/v1/login" \
+ -H  "accept: application/json" \
+ -H  "Content-Type: application/json" \
+ -d "{\"password\":\"admin\",\"username\":\"admin\"}" \
+ --insecure |
+ python -m json.tool
+
+You will get following response :
+
+ {
+    "token": "MY_TOKEN"
+ }
+
+Then use this token to ask API (here get domains config) :
```

Comment:
I propose: *Then use this token to query the API. The following example fetches the domains configuration:*